www/js/diary/infinite_scroll_trip_item.js

```diff
@@ -0,0 +1,159 @@
+
+/**
+ * A directive to display each trip within the diary view.
+ */
+
+angular.module('emission.main.diary.infscrolltripitem', [
+                                                        'emission.main.diary.infscrolllist',
+                                                        'emission.survey.multilabel.services',
+                                                        'emission.main.diary.infscrolldetail',
+                                                        'ui-leaflet', 'ng-walkthrough',
+                                                        'nvd3', 'emission.plugin.kvstore',
+                                                        'emission.services',
+                                                        'emission.config.imperial',
+                                                        'emission.plugin.logger',
+                                                        'emission.stats.clientstats',
+                                                        'emission.incident.posttrip.manual'
+])
+
+.directive("infiniteScrollTripItem", function(){
+    return{
+      restrict: 'E',
+      scope: {
+        trip: '='
+      },
+      controller: 'TripItemCtrl',
+      templateUrl: 'templates/diary/trip_list_item.html'
+    };
+  })
+
+  .controller("TripItemCtrl", function($scope, $injector, $ionicPlatform,
+                                        $state, leafletMapEvents, 
+                                        nzTour, Timeline, DiaryHelper, SurveyOptions,
+                                        Config, $ionicScrollDelegate
+                                        ){
+    console.log("Trip Item Controller called");
+    const DEFAULT_ITEM_HT = 274;
+    $scope.surveyOpt = SurveyOptions.MULTILABEL;
+    $scope.itemHt = DEFAULT_ITEM_HT;
+
+    // Added function from infiniteDiaryListCtrl
+    $scope.showDetail = function($event) {
+      $state.go("root.main.inf_scroll-detail", {
+          tripId: $scope.trip.id
+      });
```

Comment:
Did you unify the detail screens as well? I'm guessing not, in which case you should clarify that in a commit or pull request...

Reply:
@sebastianbarry you can skip reviewing this, although we should add it to the list of tasks for the pending unification.

Reply:
Added to the list, 

_**Confirming this change as complete**_